src/lib/AST/ASTVisitor.cpp

```diff
@@ -2395,6 +2637,46 @@ traverse(FriendDecl* D)
     buildFriend(I, created, D);
 }
 
+//------------------------------------------------
+// NamespaceAliasDecl
+
+void
+ASTVisitor::
+traverse(NamespaceAliasDecl* D)
+{
+    auto exp = getAsMrDocsInfo(D);
+    if( ! exp) { return; }
```

Comment:
Why the space before `!`?

Reply:
Code style, I like the negation to be visible.

`if(!lama)` -> negation is imperceptible
`if(not lama)` ->not always recommended
`if( ! lama)` -> nice! ;)

Reply:
@fpelliccioni please follow the style of the existing code